pallets/delegation/src/lib.rs

```diff
@@ -129,7 +130,11 @@ pub mod pallet {
 		>;
 		type DelegationEntityId: Parameter;
 		type DelegationNodeId: Parameter + Copy + AsRef<[u8]> + Eq + PartialEq + Ord + PartialOrd;
-		type EnsureOrigin: EnsureOrigin<Success = DelegatorIdOf<Self>, <Self as frame_system::Config>::Origin>;
+		type EnsureOrigin: EnsureOrigin<
+			Success = <Self as Config>::OriginSuccess,
+			<Self as frame_system::Config>::Origin,
+		>;
+		type OriginSuccess: CallSources<<Self as frame_system::Config>::AccountId, DelegatorIdOf<Self>>;
```

Comment:
S: should we declare and use `AccountIdOf<T>` also here?

Reply:
2f80b2d